script.js

```diff
@@ -0,0 +1,107 @@
+// "use strict";
+
+// ES5
+function Calculator(value) {
+	if (typeof(value) == 'number') {

```

Comment:
try to get into the habit of using `===`. There are very rare cases where you actually need to use the `==`